hw1/02-palindrome.js

```diff
@@ -1,3 +1,23 @@
 const elem = document.querySelector('input');
-
+const resultText = document.getElementById('resultText');
 elem.addEventListener('input', handleInput);
+
+function handleInput(){
+    let input = parseInt(elem.value);
+    if(input >= 0){
+        let reverseInput = parseInt(String(elem.value).split('').reverse().join(''));
```

Comment:
Airbnb style guide suggests recommends using const instead of let

Reply:
Hi Saurav! I have updated it to let to const and also updated the traditional functions to arrow functions. Thank you for reviewing my work!